jsonnet/jsonnet.bzl

```diff
@@ -67,18 +78,21 @@ def _setup_deps(deps):
         imports.append(dep[JsonnetLibraryInfo].imports)
         short_imports.append(dep[JsonnetLibraryInfo].short_imports)
 
-    return struct(
+    for code_file in tla_code_libraries.keys() + ext_code_libraries.keys():
+        transitive_sources.append(code_file[JsonnetLibraryInfo].transitive_jsonnet_files)
+        imports.append(code_file[JsonnetLibraryInfo].imports)
+        short_imports.append(code_file[JsonnetLibraryInfo].short_imports)
+
+    return JsonnetLibraryInfo(
         imports = depset(transitive = imports),
         short_imports = depset(transitive = short_imports),
-        transitive_sources = depset(transitive = transitive_sources, order = "postorder"),
+        transitive_jsonnet_files = depset(transitive = transitive_sources, order = "postorder"),
```

Comment:
What's the rationale for changing the name of this provider field? Unless there is a strong reason, I'd prefer it if the naming was kept as is, so that any code outside of rules_jsonnet that depends on it remains functional.

Reply:
Rationale was to align it with the `JsonnetLibraryInfo` provider as, from my read, this function is merging all of the `JsonnetLibraryInfo` from its deps together.

Happy to revert it back to keep compatibility.